src/Drupal/V8/DkanCommands.php

```diff
@@ -27,8 +27,23 @@ public function dkanTestCypress() {
 
     /**
      * Run DKAN PhpUnit Tests.
+     * 
+     * Because of a limitation of using the single `$args` array to pass all
```

Comment:
I wonder if it would be better to just create a simple proxy to phpunit inside of the container in the same way we do drush and composer? https://github.com/GetDKAN/dkan-tools/blob/master/src/Command/BasicCommands.php

Reply:
The issue with that is that `phpunit` is in defferent locations for V7 and V8.

It's `bin/phpunit` in V7 and   `$projDir}/docroot/vendor/bin/phpunit` in V8

Reply:
@ketwaroo Good point. We have a mechanism to solve that if that is the only drawback. If you look in `src\Drupal` you will see that we have a `V7` and a `V8` directories. The commands in the `V7` directory only are loaded when running in `V7` mode and viceversa. So you can have 2 phpunit commands each working differently.

Reply:
fixed in c97157faa449424fde40458ef0a61e08a4391d40